### PR TITLE
Fix pivot boot failure

### PR DESCRIPTION
**Pull request by santoshmahto7**

### CODE REVIEW

**Review by obbardc — APPROVED** [review]

**Comment by obbardc** [comment]
Looks good to me.

File: flasher/overlays/installer/usr/local/bin/installer.sh [hunk 1]
```diff
@@ -101,11 +101,25 @@ if [ ${BMAP_EXITCODE} != 0 ] ; then
   read -p "Press return to shutdown your computer..."
   shutdown -h now
 fi
+sync
+
+# wait for target device to ready
+sleep 10
```
Comment (htejun) [inline flasher/overlays/installer/usr/local/bin/installer.sh:107]: Hmm... why is this necessary? Shouldn't udevadm be able to wait for everything?

**Review by htejun — APPROVED** [review]

File: flasher/overlays/installer/usr/local/bin/installer.sh (hunk 1 above)
Comment (obbardc) [reply flasher/overlays/installer/usr/local/bin/installer.sh:107]: The sleep is belt & braces. We can remove that in a followup if things work now. cc @santoshmahto7

Comment (santoshmahto7) [reply flasher/overlays/installer/usr/local/bin/installer.sh:107]: > The sleep is belt & braces. We can remove that in a followup if things work now. cc @santoshmahto7

yes, agreed
